To: vim_dev@googlegroups.com
Subject: Patch 7.3.294
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.294 (after 7.3.289)
Problem:    Patch 289 causes more problems than it solves.
Solution:   Rever the patch untill a better solution is found.
Files:	    src/edit.c


*** ../vim-7.3.293/src/edit.c	2011-08-28 16:02:23.000000000 +0200
--- src/edit.c	2011-09-05 20:07:49.000000000 +0200
***************
*** 3367,3381 ****
      ins_bytes(compl_leader + ins_compl_len());
      compl_used_match = FALSE;
  
- #ifdef FEAT_COMPL_FUNC
-     /*
-      * To call eval 'completefunc' when leader is changed, restart completion
-      * every time.
-      */
-     if (ctrl_x_mode == CTRL_X_FUNCTION || ctrl_x_mode == CTRL_X_OMNI)
-         ins_compl_restart();
- #endif
- 
      if (compl_started)
  	ins_compl_set_original_text(compl_leader);
      else
--- 3367,3372 ----
*** ../vim-7.3.293/src/version.c	2011-09-02 14:18:14.000000000 +0200
--- src/version.c	2011-09-05 20:08:27.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     294,
  /**/

-- 
Scientists decoded the first message from an alien civilization:
        SIMPLY SEND 6 TIMES 10 TO THE 50 ATOMS OF HYDROGEN TO THE STAR
SYSTEM AT THE TOP OF THE LIST, CROSS OFF THAT STAR SYSTEM, THEN PUT
YOUR STAR SYSTEM AT THE BOTTOM OF THE LIST AND SEND IT TO 100 OTHER
STAR SYSTEMS.  WITHIN ONE TENTH GALACTIC ROTATION YOU WILL RECEIVE
ENOUGH HYDROGREN TO POWER YOUR CIVILIZATION UNTIL ENTROPY REACHES ITS
MAXIMUM!  IT REALLY WORKS!

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
